Sources/ComposableArchitecture/Dependencies/Dismiss.swift

```diff
@@ -25,7 +25,7 @@ extension DependencyValues {
 ///   func reduce(into state: inout State, action: Action) -> Effect<Action> {
 ///     switch action {
 ///     case .exitButtonTapped:
-///       return .fireAndForget { await self.dismiss() }
+///       return .run { _ in await self.dismiss() }
```

Comment:
Here, I've replaced `fireAndForget`, which has been deprecated, with `run(priority:operation:catch:fileID:line:)` to better reflect the current state of TCA.